openedx/core/djangoapps/video_config/services.py

```diff
@@ -91,3 +94,31 @@ def is_hls_playback_enabled(self, course_id: CourseKey) -> bool:
         Check if HLS playback is enabled for the course.
         """
         return HLSPlaybackEnabledFlag.feature_enabled(course_id)
+
+    def get_transcript(
+        self,
+        video_block,
+        lang: Optional[str] = None,
+        output_format: str = 'srt',
+        youtube_id: Optional[str] = None,
+    ) -> Tuple[bytes, str, str]:
```

Comment:
thanks for the type annotations!

Nit: As of python3.11, you can annotate the types without having to import anything from `typing`. Exact same effect, just a little more concise.
```suggestion
        lang: str | None = None,
        output_format: str = 'srt',
        youtube_id: str | None = None,
    ) -> tuple[bytes, str, str]:
```